### ISSUE TITLE
Capitalized non-function not triggering rule

### ISSUE THREAD

**Issue by uthbees** [ArnaudBarre/eslint-plugin-react-refresh#26]
Camel case variables are flagged correctly:
```ts
export function Component() {}
export const aa = 'a' // <--- flagged
```

But the rule seems to think that pascal case variables, no matter what they are, are components:
```ts
export function Component() {}
export const Aa = 'a' // <- not flagged!
```

This is clearly not ideal - is this possible to fix?

**Comment by ArnaudBarre** [comment]
Sorry for the late answer!
Fixed on main, I will released later this week when other issues are handled!

**Comment by ArnaudBarre** [comment]
Released in v0.4.4!
